build.gradle.kts

```diff
@@ -26,15 +26,18 @@ repositories {
 
 dependencies {
     implementation("org.springframework.boot:spring-boot-starter-web")
+    developmentOnly("org.springframework.boot:spring-boot-devtools")
+    testImplementation("org.springframework.boot:spring-boot-starter-test")
+    implementation("org.springframework.boot:spring-boot-starter-data-mongodb")
     implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
     implementation("org.jetbrains.kotlin:kotlin-reflect")
+    implementation ("io.github.microutils:kotlin-logging-jvm:2.0.11")
     compileOnly("org.projectlombok:lombok")
-    developmentOnly("org.springframework.boot:spring-boot-devtools")
     annotationProcessor("org.projectlombok:lombok")
-    testImplementation("org.springframework.boot:spring-boot-starter-test")
     implementation(kotlin("stdlib-jdk8"))
-    implementation ("io.github.microutils:kotlin-logging-jvm:2.0.11")
-    implementation("org.springframework.boot:spring-boot-starter-data-mongodb")
+    implementation("io.nats:jnats:2.16.14")
+    implementation("com.google.protobuf:protobuf-java:3.19.6")
+    implementation(project(":nats"))
```

Comment:
now sort by type of dependency (implementation, testImplementation)

Reply:
✅